tools/killswitch/src/main.rs

```diff
@@ -4,22 +4,50 @@ extern crate assert_matches;
 
 mod errors;
 mod killswitch;
-mod output;
+mod secrets;
 mod settings;
 
-use crate::killswitch::KillSwitch;
-use crate::output::Message;
-use crate::{errors::Error, output::Output, settings::Settings};
+use crate::{errors::Error, killswitch::KillSwitch, secrets::Secrets, settings::Settings};
 use clap::{ArgGroup, Parser, ValueEnum};
 use std::{
     env,
     io::{stdout, Write},
     process::exit,
+    sync::mpsc::channel,
 };
 
+/// AWS settings
+const AWS_REGION: &str = "us-west-2";
+const AWS_CREDENTIALS_PROFILE_DEVELOPMENT: &str = "nomad-xyz-dev";
+const AWS_CREDENTIALS_PROFILE_PRODUCTION: &str = "nomad-xyz-prod";
+const CONFIG_S3_BUCKET_DEVELOPMENT: &str = "nomad-killswitch-config-dev";
+const CONFIG_S3_BUCKET_PRODUCTION: &str = "nomad-killswitch-config-prod";
+const CONFIG_S3_KEY: &str = "config.yaml";
```

Comment:
naming seems off. is this a key or a filepath?

Reply:
That's how `S3` defines them. Agreed though